doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,258 @@
+
+There are Ansible [playbooks](https://galaxy.ansible.com/pbench/agent) available via Ansible Galaxy to install the `pbench-agent`, and the pieces needed (key and configuration files) to be able to send results to a server.  To use the RPMs provided above via COPR with the [playbooks](https://galaxy.ansible.com/pbench/agent), an inventory file needs to include the `fedoraproject_username` variable set to `portante`, for example:
+
+```
+...
+
+[servers:vars]
+fedoraproject_username: portante
+
+...
+```
+
+Alternatively, one can specify `fedoraproject_username` on the command line, rather than having it specified in the inventory file:
+
+    ansible-playbook -i <inventory> <playbook> -e '{fedoraproject_username: portante}'
+
+_**NOTE WELL**_: If the inventory file also has a definition for `pbench_repo_url_prefix` (which was standard practice before `fedoraproject_username` was introduced), it needs to be deleted, otherwise it will override the default repo URL and the `fedoraproject_username` change is not going to take effect.
+
+While we don't include installation instructions for the new `node-exporter` and `dcgm` tools in the published documentation, you can find a manual installation procedure for the Prometheus "node_exporter" and references to the Nvidia "DCGM" documentation in the [`agent/tool-scripts/README`](https://github.com/distributed-system-analysis/pbench/blob/b0.70/agent/tool-scripts/README.md).
+
+Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `beta`, `v0.70.0-9`, and `a7d1f0d6`.
+
+Agent
+====
+In addition to the major changes described above for this release, the following significant changes for the agent are also worth calling out specifically:
+
+ * The `pbench-fio` bench script now requires `fio-3.21` or later; see 5048a149
+
+   * You can find a [Fedora COPR](https://copr.fedorainfracloud.org/coprs/portante/pbench/) build of `fio-3.21-6.pbench` for EPEL 7 & 8
+
+ * [_**DEPRECATED**_] The `pbench-cleanup` utility command is deprecated, and will be removed in a subsequent release (see PR #1828)
+
+ * The release takes in the latest code from stockpile, including support for GPU data collection via the [Nvidia SMI](https://github.com/cloud-bulldozer/stockpile/tree/master/roles/nvidia_smi) role
+
+ * The new `dcgm` tool requires Python 2, an Nvidia based install which might conflict with the Pbench Agent's Python 3 operational requirement in some cases
+
+Server
+====
+Initial support for moving results to the pbench-server via HTTP `PUT` operations.
+
+Web Server
+====
+There are no changes and no new `web-server` RPMs have been produced.
+
+Pbench Dashboard
+====
+The development of the dashboard is not tracked in these release notes. The dashboard has been moved into its own [git repo](https://github.com/distributed-system-analysis/pbench-dashboard).
```

Comment:
This gets complicated, since we haven't "released" any sort of user-enabled server, and the old `pbench-dashboard` is still in use for 0.69 ... but did you intend (and do we want) to retain this reference?

Reply:
I'll be reworking this.